A43-sdk-authorization.md

```diff
@@ -0,0 +1,905 @@
+A43: SDK authorization
+
+* Author(s): [Ashitha Santhosh](https://github.com/ashithasantosh)
+* Approver: [Doug Fawley](https://github.com/dfawley),
+            [Eric Anderson](https://github.com/ejona86),
+            [Mark Roth](https://github.com/markdroth),
+            [Sanjay Pujare](https://github.com/sanjaypujare),
+            [Yihua Zhang](https://github.com/yihuazhang)
+* Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+* Implemented in: <language, ...>
+* Last updated: 2021-07-19
+* Discussion at: (filled after thread exists)
+
+## Abstract
+
+gRPC is currently implementing an open-source standard authorization solution
+based on Envoy RBAC. This proposal discusses how this framework would be made 
+available to non-xDS OSS gRPC users to perform per-RPC authorization checks.
+
+## Background
+
+Authentication verifies the authority of the request. Authorization ensures the 
```

Comment:
Updated the line. Since authentication is not a strict requirement for authorization, I reworded it as -
Authorization ensures that that requestor has sufficient access to make a particular request.